<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>YesNoDialog</class>
 <widget class="QDialog" name="YesNoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>308</width>
    <height>136</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>خروج</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/icons/resources/icons/question.png</normaloff>:/icons/resources/icons/question.png</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <property name="locale">
   <locale language="Persian" country="Iran"/>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="VLayoutMain">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetNoConstraint</enum>
   </property>
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="HLayoutMessage">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="lblIcon">
       <property name="minimumSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>B Nazanin</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resource.qrc">:/icons/resources/icons/question.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblMessage">
       <property name="font">
        <font>
         <family>B Nazanin</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>آیا می‌خواهید از برنامه خارج شوید؟</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="HLayoutButtons">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnYes">
       <property name="font">
        <font>
         <family>B Nazanin</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>بله</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnNo">
       <property name="font">
        <font>
         <family>B Nazanin</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>خیر</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
